pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1832,23 +1831,67 @@ func addNDotsOption(resolvFilePath string) error {
 		return fmt.Errorf("ResolvConfPath %q does not exist", resolvFilePath)
 	}
 
-	glog.V(4).Infof("DNS ResolvConfPath exists: %s. Will attempt to add ndots option: %s", resolvFilePath, ndotsDNSOption)
+	curResolvFileContent, err := getFileContent(resolvFilePath)
```

Comment:
why does this matter - you're not saving any of it at this point.   Save the IO and just generate a new file without looking?